main.go

```diff
@@ -31,9 +31,11 @@ func main() {
 	// Create a new logger which is used by all packages.
 	var newLogger micrologger.Logger
 	{
-		loggerConfig := micrologger.DefaultConfig()
-		loggerConfig.IOWriter = os.Stdout
-		newLogger, err = micrologger.New(loggerConfig)
+		c := micrologger.DefaultConfig()
+
+		c.IOWriter = os.Stdout
+
+		newLogger, err = micrologger.New(c)
```

Comment:
I am only renaming here for shorter vars and not applying the latest structure advances because the default config has to be removed which implies managing the defaulting inside the constructor. This is yak shaving I do not want to do now.